### ISSUE TITLE
[EPIC] Source AppsFlyer: Enable in Cloud and Certify to Beta

### ISSUE THREAD

**Issue by lazebnyi** [airbytehq/airbyte#24306]

**Comment by igrankova** [comment]
@YowanR, we are blocked on a paid raw data reports functionality: 0.56 $ per conversion - [Growth plan](https://www.appsflyer.com/pricing/)
Conversion is the click that brought engages or app installs, as we understand. As we have an app which is not live and we'll have no conversions, we think we can switch a plan on and use it for free for enabling on Cloud. But further if we need to seed it, than it could charge us. I also asked a support team for sandbox or dev access to the raw data reports, waiting for response. 
Should we proceed with this connector in case it's a paid feature for us?
cc @artem1205

**Comment by YowanR** [comment]
This connector is not in scope for Q1B so let's skip it, please. We'll revisit once we decide to tackle it
